### ISSUE TITLE
Reduce verbosity of output when performing:  podman compose up

### ISSUE THREAD

**Issue by jwmatthews** [konveyor/kai#228]
When a new user runs `podman compose up` they will see:
1) Potential error message: `psql: error: could not connect to server: Connection refused`
2) ~thousands+ of lines of debug info from 'load-data'

Example:

```
$ time podman compose up
>>>> Executing external compose provider "/usr/local/bin/docker-compose". Please refer to the documentation for details. <<<<

WARN[0000] The "POSTGRES_USER" variable is not set. Defaulting to a blank string. 
[+] Running 2/0
 ⠿ Container kai-kai_db-1  Created                                                                                                                   0.0s
 ⠿ Container kai-kai-1     Recreated                                                                                                                 0.0s
Attaching to kai-kai-1, kai-kai_db-1
kai-kai_db-1  | 
kai-kai_db-1  | PostgreSQL Database directory appears to contain a database; Skipping initialization
kai-kai_db-1  | 
kai-kai_db-1  | 2024-07-03 17:07:52.122 UTC [1] LOG:  starting PostgreSQL 15.7 (Debian 15.7-1.pgdg120+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
kai-kai_db-1  | 2024-07-03 17:07:52.122 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
kai-kai_db-1  | 2024-07-03 17:07:52.122 UTC [1] LOG:  listening on IPv6 address "::", port 5432
kai-kai-1     | psql: error: could not connect to server: Connection refused
kai-kai-1     | 	Is the server running on host "kai_db" (10.89.1.11) and accepting
kai-kai-1     | 	TCP/IP connections on port 5432?
kai-kai-1     | load-data has not run yet, starting ...
kai-kai-1     | Fetching examples
kai-kai_db-1  | 2024-07-03 17:07:52.124 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
kai-kai_db-1  | 2024-07-03 17:07:52.128 UTC [25] LOG:  database system was shut down at 2024-07-03 17:07:44 UTC
kai-kai_db-1  | 2024-07-03 17:07:52.131 UTC [1] LOG:  database system is ready to accept connections
kai-kai-1     | Cloning kitchensink...
kai-kai-1     | *** Exiting since 'sample_repos/kitchensink' already exists
kai-kai-1     | *** Delete 'sample_repos' and rerun this script
kai-kai-1     | INFO - 2024-07-03 17:07:54,038 - [             psql.py:55   -             __init__()] - Connected to the PostgreSQL server.
kai-kai-1     | INFO - 2024-07-03 17:07:54,078 - [   incident_store.py:86   - load_reports_from_directory()] - Loading incident store with applications: ['bmt', 'cmt', 'ejb-remote', 'ejb-security', 'greeter', 'helloworld-mdb', 'kitchensink', 'tasks-qute', 'ticket-monster']
.. .
<snip>
...
kai-kai-1     | DEBUG - 2024-07-03 17:09:09,659 - [             psql.py:311  -          load_report()] - diff --git a/monolith/src/main/java/org/jboss/examples/ticketmonster/rest/TicketPriceEndpoint.java b/monolith/src/main/java/org/jboss/examples/ticketmonster/rest/TicketPriceEndpoint.java
kai-kai-1     | index e96e958..10f75eb 100644
kai-kai-1     | --- a/monolith/src/main/java/org/jboss/examples/ticketmonster/rest/TicketPriceEndpoint.java
kai-kai-1     | +++ b/monolith/src/main/java/org/jboss/examples/ticketmonster/rest/TicketPriceEndpoint.java
kai-kai-1     | @@ -1,38 +1,39 @@
kai-kai-1     |  package org.jboss.examples.ticketmonster.rest;
kai-kai-1     |  
kai-kai-1     | +import jakarta.enterprise.context.ApplicationScoped;
kai-kai-1     | +import jakarta.inject.Inject;
kai-kai-1     | +import jakarta.persistence.EntityManager;
kai-kai-1     | +import jakarta.persistence.NoResultException;
kai-kai-1     | +import jakarta.persistence.OptimisticLockException;
kai-kai-1     | +import jakarta.persistence.PersistenceContext;
kai-kai-1     | +import jakarta.persistence.TypedQuery;
kai-kai-1     | +import jakarta.ws.rs.Consumes;
kai-kai-1     | +import jakarta.ws.rs.DELETE;
kai-kai-1     | +import jakarta.ws.rs.GET;
kai-kai-1     | +import jakarta.ws.rs.POST;
kai-kai-1     | +import jakarta.ws.rs.PUT;
kai-kai-1     | +import jakarta.ws.rs.Path;
kai-kai-1     | +import jakarta.ws.rs.PathParam;
kai-kai-1     | +import jakarta.ws.rs.Produces;
kai-kai-1     | +import jakarta.ws.rs.QueryParam;
kai-kai-1     | +import jakarta.ws.rs.core.Response;
kai-kai-1     | +import jakarta.ws.rs.core.Response.Status;
kai-kai-1     | +import jakarta.ws.rs.core.UriBuilder;
kai-kai-1     | +import org.jboss.examples.ticketmonster.model.TicketPrice;
kai-kai-1     | +import org.jboss.examples.ticketmonster.rest.dto.TicketPriceDTO;
kai-kai-1     | +
kai-kai-1     |  import java.util.ArrayList;
kai-kai-1     |  import java.util.List;
kai-kai-1     |  
kai-kai-1     | -import javax.ejb.Stateless;
kai-kai-1     | -import javax.persistence.EntityManager;
kai-kai-1     | -import javax.persistence.NoResultException;
kai-kai-1     | -import javax.persistence.OptimisticLockException;
kai-kai-1     | -import javax.persistence.PersistenceContext;
kai-kai-1     | -import javax.persistence.TypedQuery;
kai-kai-1     | -import javax.ws.rs.Consumes;
kai-kai-1     | -import javax.ws.rs.DELETE;
kai-kai-1     | -import javax.ws.rs.GET;
kai-kai-1     | -import javax.ws.rs.POST;
kai-kai-1     | -import javax.ws.rs.PUT;
kai-kai-1     | -import javax.ws.rs.Path;
kai-kai-1     | -import javax.ws.rs.PathParam;
kai-kai-1     | -import javax.ws.rs.Produces;
kai-kai-1     | -import javax.ws.rs.QueryParam;
kai-kai-1     | -import javax.ws.rs.core.Response;
kai-kai-1     | -import javax.ws.rs.core.Response.Status;
kai-kai-1     | -import javax.ws.rs.core.
...
```

**Comment by jwmatthews** [comment]
I'm working on a change that will:
1) use [wait-for-it.sh](https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh) to wait for postgres to be running
2) reduce verbosity of load-data

**Comment by jwmatthews** [comment]
Piece #1 of "psql: error: could not connect to server: Connection refused" is fixed by #225

**Comment by jwmatthews** [comment]
This has been addressed with a few PRs.

Confirmed looking better with https://github.com/konveyor/kai/releases/tag/demo.2024-07-22
